.travis.yml

```diff
@@ -57,5 +59,15 @@ after_success:
         brew install lcov;
       fi
     - lcov --directory test --base-directory ../include/meshio/ --capture --no-external --output-file coverage.info
+    - genhtml coverage.info --output-directory ./docs/html/coverage
     - lcov --remove coverage.info '/usr*' -o coverage.info
     - coveralls-lcov coverage.info
+
+deploy:
+    provider: pages
+    skip_cleanup: true
+    local_dir: ./build/docs/html
+    github_token: $GH_REPO_TOKEN
+    on:
+        branch: master
+        os: osx
```

Comment:
have you tested the docs deployment on your fork ?

Reply:
Yes,

* https://github.com/acrlakshman/meshio-test
* https://acrlakshman.github.io/meshio-test/

Reply:
FYI- We can have this on master for now, but at some point, we should change this to only on tag creation i.e. release.